### PR TITLE
Reformatting MulticlassClassification samples to width 85

### PR DESCRIPTION
**Pull request by sierralee51**
Guidelines followed:
-85 characters per line
-Use 4 spaces for indentation
-Dot and open parentheses stay on same line as function
-If not a preexisting line under line that we break, add an extra line after it
-Don't indent comments
-Don't break a comment if it represents output
-Don't break links
-If applicable, break right before $
-Keep math op together

Fix for issue #3478
*There are a couple exceptions to the 85 char. limit that have been approved by Natalie

### CODE REVIEW

**Comment by sierralee51** [comment]
Just noticed that, like last time, there are some extra spaces before comments that don't show up on VisualStudio. Fixing them through GitHub right now.

**Comment by sierralee51** [comment]
> Just noticed that, like last time, there are some extra spaces before comments that don't show up on VisualStudio. Fixing them through GitHub right now.

In fixing these extra spaces before comments, I also realized that all my indentations were messed up. The extra spaces in my PR should be fixed, and I am now investigating the issue of extra whitespace in GitHub, as I know that Sayan and I both encountered this despite using spaces rather than tabs. A quick search has shown that this is a relatively common isssue, so I'll be looking into ways to prevent this in the future.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs [hunk 1]
```diff
@@ -10,39 +10,50 @@ public static class LbfgsMaximumEntropy
     {
         public static void Example()
         {
-            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
-            // as a catalog of available operations and as the source of randomness.
-            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            // Create a new context for ML.NET operations. It can be used for
+	    // exception tracking and logging, as a catalog of available operations
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs]: >	  [](start = 0, length = 2)

why do you have these tabs?

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LightGbm.cs [hunk 2]
```diff
@@ -8,43 +8,55 @@ namespace Samples.Dynamic.Trainers.MulticlassClassification
...
+	        .LoadFromEnumerable(GenerateRandomDataPoints(500, seed: 123));
 
             // Run the model on test data set.
             var transformedTestData = model.Transform(testData);
 
             // Convert IDataView object to a list.
-            var predictions = mlContext.Data.CreateEnumerable<Prediction>(transformedTestData, reuseRowObject: false).ToList();
+            var predictions = mlContext.Data
+                .CreateEnumerable<Prediction>(transformedTestData,
+		reuseRowObject: false).ToList();
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LightGbm.cs]: >		 [](start = 0, length = 2)

tabs. please convert tabs to spaces. 
[https://blogs.msdn.microsoft.com/zainnab/2010/03/14/how-to-convert-tabs-to-spaces-and-vice-versa/](https://blogs.msdn.microsoft.com/zainnab/2010/03/14/how-to-convert-tabs-to-spaces-and-vice-versa/)

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/MulticlassClassification.ttinclude [hunk 3]
```diff
@@ -97,13 +119,17 @@ namespace Samples.Dynamic.Trainers.MulticlassClassification
...
+                    Features = Enumerable.Repeat(label, 20)
+                        .Select(x => randomFloat() + label * 0.2f).ToArray()
+
                 };
             }
         }
 
-        // Example with label and 20 feature values. A data set is a collection of such examples.
+        // Example with label and 20 feature values. A data set is a collection of
+	// such examples.
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/MulticlassClassification.ttinclude]: Align the //

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/NaiveBayes.cs [hunk 4]
```diff
@@ -9,44 +9,57 @@ namespace Samples.Dynamic.Trainers.MulticlassClassification
...
+                .LoadFromEnumerable(GenerateRandomDataPoints(500, seed: 123));
 
             // Run the model on test data set.
             var transformedTestData = model.Transform(testData);
 
             // Convert IDataView object to a list.
-            var predictions = mlContext.Data.CreateEnumerable<Prediction>(transformedTestData, reuseRowObject: false).ToList();
+            var predictions = mlContext.Data
+                .CreateEnumerable<Prediction>(transformedTestData,
+		reuseRowObject: false).ToList();
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/NaiveBayes.cs]: >reuseRowObject: false).ToList(); [](start = 2, length = 32)

this needs to be indented after .CreateEnumerable with 4 splaces.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/NaiveBayes.cs [hunk 5]
```diff
@@ -9,44 +9,57 @@ namespace Samples.Dynamic.Trainers.MulticlassClassification
...
 
             // Look at 5 predictions
             foreach (var p in predictions.Take(5))
-                Console.WriteLine($"Label: {p.Label}, Prediction: {p.PredictedLabel}");
+                Console.WriteLine($"Label: {p.Label}, " + 
+		    $"Prediction: {p.PredictedLabel}");
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/NaiveBayes.cs]: >$"Prediction: {p.PredictedLabel}"); [](start = 6, length = 35)

same here but after Console.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/NaiveBayes.cs [hunk 6]
```diff
@@ -56,7 +69,9 @@ public static void Example()
             //   Label: 3, Prediction: 3
 
             // Evaluate the overall metrics
-            var metrics = mlContext.MulticlassClassification.Evaluate(transformedTestData);
+            var metrics = mlContext.MulticlassClassification
+	        .Evaluate(transformedTestData);
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/NaiveBayes.cs]: >.Evaluate(transformedTestData); [](start = 9, length = 31)

indent

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/NaiveBayes.tt [hunk 7]
```diff
@@ -9,13 +9,16 @@ bool CacheData = false;
...
 string Comments= @"
         // Naive Bayes classifier is based on Bayes' theorem. 
-        // It assumes independence among the presence of features in a class even though they may be dependent on each other.
-        // It is  a multi-class trainer that accepts binary feature values of type float, i.e., feature values are either true or false.
-        // Specifically a feature value greater than zero is treated as true, zero or less is treated as false.";
+        // It assumes independence among the presence of features in a class even
+        // though they may be dependent on each other. It is  a multi-class trainer
+	// that accepts binary feature values of type float, i.e., feature values
+	// are either true or false. Specifically a feature value greater than zero
+	// is treated as true, zero or less is treated as false.";
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/NaiveBayes.tt]: Alight all //

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/PairwiseCoupling.cs [hunk 8]
```diff
@@ -10,39 +10,50 @@ public static class PairwiseCoupling
...
             // Define the trainer.
             var pipeline =
                     // Convert the string labels into key types.
                     mlContext.Transforms.Conversion.MapValueToKey("Label")
-                    // Apply PairwiseCoupling multiclass meta trainer on top of binary trainer.
-                    .Append(mlContext.MulticlassClassification.Trainers.PairwiseCoupling(mlContext.BinaryClassification.Trainers.SdcaLogisticRegression()));
+                    // Apply PairwiseCoupling multiclass meta trainer on top of
+		    // binary trainer.
+                    .Append(mlContext.MulticlassClassification.Trainers
+		        .PairwiseCoupling(
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/PairwiseCoupling.cs]: >.PairwiseCoupling( [](start = 10, length = 18)

indent after .Append with 4 spaces

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/PairwiseCoupling.cs [hunk 9]
```diff
@@ -10,39 +10,50 @@ public static class PairwiseCoupling
...
+			mlContext.BinaryClassification.Trainers.SdcaLogisticRegression()));
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/PairwiseCoupling.cs]: >mlContext.BinaryClassification.Trainers.SdcaLogisticRegression())); [](start = 3, length = 67)

indent after .PairwiseCoupling with 4 spaces.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/PairwiseCoupling.cs [hunk 10]
```diff
@@ -10,39 +10,50 @@ public static class PairwiseCoupling
...
 
             // Train the model.
             var model = pipeline.Fit(trainingData);
 
-            // Create testing data. Use different random seed to make it different from training data.
-            var testData = mlContext.Data.LoadFromEnumerable(GenerateRandomDataPoints(500, seed: 123));
+            // Create testing data. Use different random seed to make it different
+	    // from training data.
+            var testData = mlContext.Data
+	        .LoadFromEnumerable(GenerateRandomDataPoints(500, seed: 123));
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/PairwiseCoupling.cs]: >.LoadFromEnumerable(GenerateRandomDataPoints(500, seed: 123)); [](start = 9, length = 62)

indent after var with 4 spaces.

**Comment by codemzs** [comment]
It seems the indentation needs to be fixed. My comments apply at many places where I have not commented.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs [hunk 11]
```diff
@@ -10,39 +10,49 @@ public static class LbfgsMaximumEntropy
...
+            // consumable by ML.NET API.
             var trainingData = mlContext.Data.LoadFromEnumerable(dataPoints);
 
             // Define the trainer.
             var pipeline =
-                    // Convert the string labels into key types.
-                    mlContext.Transforms.Conversion.MapValueToKey(nameof(DataPoint.Label))
-                    // Apply LbfgsMaximumEntropy multiclass trainer.
-                    .Append(mlContext.MulticlassClassification.Trainers.LbfgsMaximumEntropy());
+            // Convert the string labels into key types.
```
Comment (sharwell) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs]: 📝 Line should be indented by 4 more spaces

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs [hunk 12]
```diff
@@ -10,39 +10,49 @@ public static class LbfgsMaximumEntropy
...
+                mlContext.Transforms.Conversion
+                .MapValueToKey(nameof(DataPoint.Label))
```
Comment (sharwell) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs]: 📝 Line should be indented by 4 more spaces

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs [hunk 13]
```diff
@@ -10,39 +10,49 @@ public static class LbfgsMaximumEntropy
...
+            // Apply LbfgsMaximumEntropy multiclass trainer.
+                .Append(mlContext.MulticlassClassification.Trainers
+                .LbfgsMaximumEntropy());
```
Comment (sharwell) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs]: 1. Wrap the entire argument if possible (before `mlContext`)
2. Indent this line an additional 4 spaces (8 total from current location)

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs [hunk 14]
```diff
@@ -10,39 +10,49 @@ public static class LbfgsMaximumEntropy
...
+                .LoadFromEnumerable(GenerateRandomDataPoints(500, seed: 123));
 
             // Run the model on test data set.
             var transformedTestData = model.Transform(testData);
 
             // Convert IDataView object to a list.
-            var predictions = mlContext.Data.CreateEnumerable<Prediction>(transformedTestData, reuseRowObject: false).ToList();
+            var predictions = mlContext.Data
+                .CreateEnumerable<Prediction>(transformedTestData,
+                reuseRowObject: false).ToList();
```
Comment (sharwell) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs]: 📝 This is difficult to read. See preferred form in https://github.com/dotnet/machinelearning/pull/3941#discussion_r299511862

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs [hunk 15]
```diff
@@ -10,39 +10,49 @@ public static class LbfgsMaximumEntropy
...
 
             // Look at 5 predictions
             foreach (var p in predictions.Take(5))
-                Console.WriteLine($"Label: {p.Label}, Prediction: {p.PredictedLabel}");
+                Console.WriteLine($"Label: {p.Label}, " + 
+                    $"Prediction: {p.PredictedLabel}");
```
Comment (sharwell) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs]: 📝 Multi-line body of `foreach` needs braces

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs [hunk 16]
```diff
@@ -72,8 +84,11 @@ public static void Example()
             //  Precision ||0.9308 |0.9593 |0.8580 |
         }
 
-        // Generates random uniform doubles in [-0.5, 0.5) range with labels 1, 2 or 3.
-        private static IEnumerable<DataPoint> GenerateRandomDataPoints(int count, int seed=0)
+        // Generates random uniform doubles in [-0.5, 0.5)
+        // range with labels 1, 2 or 3.
+        private static IEnumerable<DataPoint> GenerateRandomDataPoints(int count,
+            int seed=0)
```
Comment (sharwell) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.cs]: 📝 Wrap no arguments, or wrap all arguments with one argument per line

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.tt [hunk 17]
```diff
@@ -8,7 +8,8 @@ string TrainerOptions = null;
 string OptionsInclude = "";
 string Comments = "";
 bool CacheData = false;
-string DataGenerationComments= "// Generates random uniform doubles in [-0.5, 0.5) range with labels 1, 2 or 3.";
+string DataGenerationComments= "// Generates random uniform doubles in [-0.5, 0.5)"
+    + "\n        // range with labels 1, 2 or 3.";
```
Comment (sharwell) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropy.tt]: 📝 Place `\n` at the end of the preceding line

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropyWithOptions.cs [hunk 18]
```diff
@@ -32,27 +34,33 @@ public static void Example()
 
             // Define the trainer.
             var pipeline = 
-			        // Convert the string labels into key types.
-                    mlContext.Transforms.Conversion.MapValueToKey("Label")
-                    // Apply LbfgsMaximumEntropy multiclass trainer.
-                    .Append(mlContext.MulticlassClassification.Trainers.LbfgsMaximumEntropy(options));
-			
+            // Convert the string labels into key types.
```
Comment (sharwell) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropyWithOptions.cs]: 📝 Needs 4 more spaces

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropyWithOptions.cs [hunk 19]
```diff
@@ -32,27 +34,33 @@ public static void Example()
...
+                mlContext.Transforms.Conversion.MapValueToKey("Label")
+            // Apply LbfgsMaximumEntropy multiclass trainer.
+                .Append(mlContext.MulticlassClassification.Trainers
+                .LbfgsMaximumEntropy(options));
```
Comment (sharwell) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropyWithOptions.cs]: 📝 See earlier comment regarding this wrapping

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LightGbmWithOptions.cs [hunk 20]
```diff
@@ -66,7 +75,9 @@ public static void Example()
             //   Label: 3, Prediction: 3
 
             // Evaluate the overall metrics
-            var metrics = mlContext.MulticlassClassification.Evaluate(transformedTestData);
+            var metrics = mlContext.MulticlassClassification
+                .Evaluate(transformedTestData);
+
```
Comment (RadicalRayan) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LightGbmWithOptions.cs]: empty line

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/OneVersusAll.cs [hunk 21]
```diff
@@ -10,39 +10,50 @@ public static class OneVersusAll
...
-                    // Apply OneVersusAll multiclass meta trainer on top of binary trainer.
-                    .Append(mlContext.MulticlassClassification.Trainers.OneVersusAll(mlContext.BinaryClassification.Trainers.SdcaLogisticRegression()));
+                // Convert the string labels into key types.
+                mlContext.Transforms.Conversion.MapValueToKey("Label")
+                // Apply OneVersusAll multiclass meta trainer on top of
+                // binary trainer.
+                .Append(mlContext.MulticlassClassification.Trainers
+                .OneVersusAll(
+                mlContext.BinaryClassification.Trainers.SdcaLogisticRegression()));
+
```
Comment (RadicalRayan) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/OneVersusAll.cs]: empty line

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/PairwiseCoupling.cs [hunk 22]
```diff
@@ -10,39 +10,50 @@ public static class PairwiseCoupling
...
-                    // Apply PairwiseCoupling multiclass meta trainer on top of binary trainer.
-                    .Append(mlContext.MulticlassClassification.Trainers.PairwiseCoupling(mlContext.BinaryClassification.Trainers.SdcaLogisticRegression()));
+                // Convert the string labels into key types.
+                mlContext.Transforms.Conversion.MapValueToKey("Label")
+                // Apply PairwiseCoupling multiclass meta trainer on top of
+                // binary trainer.
+                .Append(mlContext.MulticlassClassification.Trainers
+                .PairwiseCoupling(
+                mlContext.BinaryClassification.Trainers.SdcaLogisticRegression()));
+
```
Comment (RadicalRayan) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/PairwiseCoupling.cs]: empty line

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropyWithOptions.tt [hunk 23]
```diff
@@ -12,7 +12,9 @@ string TrainerOptions = @"LbfgsMaximumEntropyMulticlassTrainer.Options
 
 string OptionsInclude = "using Microsoft.ML.Trainers;";
 string Comments = "";
-string DataGenerationComments= "// Generates random uniform doubles in [-0.5, 0.5) range with labels 1, 2 or 3.";
+string DataGenerationComments= "// Generates random uniform doubles in [-0.5, 0.5)"
+    + "\n        // range with labels 1, 2 or 3.";
```
[the remark this answers is not in the document]
Comment (sierralee51) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LbfgsMaximumEntropyWithOptions.tt]: That isn't a comment. It's part of the string

**Pull request merged**